### PR TITLE
Explain wildcards with testable examples

### LINKED ISSUE #874: Ep3: Give real example for `?` wildcard usage
Introducing the `*` wildcard has some real examples which can be tried by the user, and demoed by the trainer. 

However, the `?` examples are all hypothetical, and/or a bit cumbersome e.g. `p*.p?*`.
Perhaps something like the differences between 
```
ls *thane.pdb
ls ?thane.pdb
ls ??thane.pdb
```
would make this section more effective, so that learners can see a demo and try it for themselves? Introducing `??` before the first exercise on wildcards might  also make the exercise easier to follow?

I can put in a PR if you agree.

### PR DESCRIPTION
**Pull request by gcapes**
Fix #874

### CODE REVIEW

**Comment by gdevenyi** [comment]
Looks great, just one small formatting quibble.

**Comment by gcapes** [comment]
Sure - what's the quibble?

**Comment by gdevenyi** [comment]
Just the double space I highlighted in the review.

**Comment by gcapes** [comment]
Sorry if I've missed something obvious, but where is the review? I don't see any comments.

**Comment by gdevenyi** [comment]
Ooops! It seems that github holds the review in a hidden state until I click "request changes" I expected you to be able to see them as I saved them!

File: _episodes/03-create.md [hunk 1]
```diff
@@ -633,23 +633,20 @@ Oftentimes one needs to copy or move several files at once. This can be done by
 
 > ## Wildcards
 >
-> `*` is a **wildcard**. It matches zero or more
-> characters, so `*.pdb` matches `ethane.pdb`, `propane.pdb`, and every
+> `*` is a **wildcard**, which matches zero or more  characters.
```
[the remark this answers is not in the document]
Comment (gcapes) [reply _episodes/03-create.md]: Fixed.

**Pull request merged**